exampleSite/content/member.md

```diff
@@ -0,0 +1,201 @@
++++
+fragment = "member"
+#disabled = false
+date = "2017-10-17"
+lastmod = "2017-10-17"
+weight = 900
+#background = "dark"
+selfLink = "member-fragment"
+
+title = "Member Fragment"
+subtitle = "Highly motivated Gophers"
+
+[[member]]
```

Comment:
Still needs to be split off into it's own member file.

Reply:
In that case, having the member as a page bundle and several members listed as separate files.

Reply:
Yes. The member files would then use the subfolder idea and the demo code posted above.